mmv1/third_party/terraform/services/container/node_config.go.tmpl

```diff
@@ -609,8 +609,9 @@ func schemaNodeConfig() *schema.Schema {
 								Description:  `Control the CPU management policy on the node.`,
 							},
 							"cpu_cfs_quota": {
-								Type:     schema.TypeBool,
-								Optional: true,
+								Type:        schema.TypeBool,
+								Optional:    true,
+								Default:     true,
```

Comment:
In order to not make this a breaking change, the `default:true` needs to be removed here. But if it's removed I don't think this PR is fixing any particular issue.